.changeset/bitter-actors-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+'@mysten/seal': patch
+---
+
+Check scalar encoding in BLS to allow using newer versions of noble/curves where the encoding can vary.
```

Comment:
```suggestion
Force scalar encoding in BLS to big endian since newer versions >=1.9.6 of noble/curves changed the default encoding to little endian.
Encryptions created by previous versions of Seal SDK and with noble/curves versions >=1.9.6 might fail `decrypt()` if called with `checkShareConsistency=true`, and might not be decryptable by the onchain `decrypt` function.
```